src/Rasterization/Path/PathApproximator.php

```diff
@@ -35,26 +35,26 @@ class PathApproximator
     /**
      * @var BezierApproximator $bezier The singleton bezier approximator.
      */
-    private static $bezier;
+    private static BezierApproximator $bezier;
```

Comment:
```suggestion
    private static ?BezierApproximator $bezier;
```

Reply:
I do think an [unitialized state](https://php.watch/versions/7.4/typed-properties#uninitialized) is semantically more correct here. 

It ensures an error if some method tries to accede it directly (without using the getter)

Reply:
Absolutely. I did not know this was a thing; thanks for teaching me something new!

Reply:
You're very welcome :)